test/integration/nql_knex.test.js

```diff
@@ -0,0 +1,28 @@
+const utils = require('../utils');
+const nql = require('../../lib/nql');
+
+const knex = utils.db.client;
+
+/**
+ * The purpose of this file is to prove that NQL
+ * is not just transformed to mongo queries correctly
+ * but that this can be used in real world settings to query SQL databases
+ */
+
+describe.skip('Integration with Knex', () => {
```

Comment:
why is this skipped?

Reply:
my fault, committed this too quick :see_no_evil: